src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +141,33 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
+    
+    @Test
+	public void getTagsToExcludeFromChart_defaultConfiguration_ReturnsEmptyList() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        Pattern[] patterns = configuration.getTagsToExcludeFromChart();
+
+        // then
+        assertThat(patterns).isEmpty();
+	}
+    
+    @Test
+	public void getTagsToExcludeFromChart_addPatterns_ReturnsListWithAllPatterns() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        Pattern somePattern = Pattern.compile("");
+        Pattern anotherPattern = Pattern.compile("");
```

Comment:
so what are you excluding now? This case is not fully clear to me, maybe rename variables and provide more meangful example?